main.cpp

```diff
@@ -299,6 +353,8 @@ struct PizzaStore
     int addToppings(std::string toppingName, bool maxedOutToppings = false);
     // hire or fire employee
     bool hireOrFireEmployee(std::string employeeName, int totalStaffMembers);
+    // new member function
+    PizzaStore getPizzaStore(int threshold, int startingValue);
```

Comment:
a Pizza Store instance will return another Pizza Store? 
that doesn't make much sense.
This is a member function, so the only way to call it is by using a Pizza Store instance...
- [ ] revise